tools/windows_limit_memory.py

```diff
@@ -670,7 +670,7 @@ def main(args: argparse.Namespace) -> int:
         proc_limiter.assign_process_to_job()
         proc_limiter.limit_process_memory(args.memory)
         if not proc_limiter.wait_for_job():
-            input("Press <enter> to exit this script")
+            eval(input("Press <enter> to exit this script"))
```

Comment:
hmm

Reply:
(script author) Yep, it's a full python 3 script, it work as it, as mentioned in the module docstring (and the shebang). No need to eval on `input`.